migrations/2_deploy_contracts.js

```diff
@@ -1 +1,12 @@
-module.exports = async (deployer) => { };
+const Property = artifacts.require("./Property.sol");
+const Market = artifacts.require("./Market.sol");
+
+module.exports = async (deployer) => {
+  await deployer.deploy(Property);
+
+  // interval = 3600 (1 hour)
+  // taxRatePerInterval = 114155 (10% / year)
```

Comment:
we need to transfer ```Property``` ownership to the ```Market``` after it has been deployed. I think Zeppelin has that stupid ```acceptOwnership``` function tho.